### ISSUE TITLE
Use this repo as a directory to other project repos?

### ISSUE THREAD

**Issue by Frijol** [tessel/project#355]
## Proposal
Add a new file to this repo which gives organized links (with brief descriptions) of each currently active repo in the Tessel org

## Why
* I am always forgetting which repo contains our devops information
* Having tesselwww and tessel.io as repos is confusing because it's not immediately obvious what is transitional & what's active
* We've just deprecated hardware-modules, but for a while it was the secret canonical source of module-related information.

*In short, our org has pages of repos, and it is hard to know where to look first.* I have trouble, and I made a bunch of these repos! Imagine how confusing this is to people who haven't been around this org for years.

## Drawbacks
Keeping things up to date. Duplicating information. The usual stuff.

## Thoughts on this?

**Comment by Frijol** [comment]
Directory organization idea:

* Product: t2-cli, t2-firmware, tessel-openwrt, tessel.io, all of the modules
* Project dev tools/product support (separate these?): rust-compilation-server (?), t2-build, pre-gypify
* Tessel Project meta: project, logos, rfcs
* Pre-release projects: reach, tessel-rust, tesselwww
